aliyun/src/test/java/org/apache/iceberg/aliyun/oss/TestOSSInputFile.java

```diff
@@ -61,13 +61,13 @@ public void testReadFile() throws Exception {
   @Test
   public void testOSSInputFile() {
     OSSURI uri = randomURI();
-    AssertHelpers.assertThrows(
-        "File length should not be negative",
-        ValidationException.class,
-        "Invalid file length",
-        () ->
-            new OSSInputFile(
-                ossClient().get(), uri, aliyunProperties, -1, MetricsContext.nullMetrics()));
+    Assertions.assertThatThrownBy(
+            () ->
+                new OSSInputFile(
+                    ossClient().get(), uri, aliyunProperties, -1, MetricsContext.nullMetrics()),
+            "File length should not be negative")
```

Comment:
nit: I would probably remove the description `File length should not be negative` from here as it doesn't add any value. However in other places in the codebase the description might be useful. 
Going forward it would be good to double-check if having the description makes sense

Reply:
> nit: I would probably remove the description `File length should not be negative` from here as it doesn't add any value. However in other places in the codebase the description might be useful. Going forward it would be good to double-check if having the description makes sense

Thank you for your comments, I will check it before modifying.